guides/common/modules/snip_note-imported-project-server.adoc

```diff
@@ -0,0 +1,4 @@
+[NOTE]
+====
+On the imported {ProjectServer}, the `/var/lib/pulp/imports` directory will get extracted to `/var/lib/pulp/` and the `/var/lib/pulp/imports` directory can be cleared out after the successful import.
```

Comment:
```suggestion
On the importing {ProjectServer}, the `/var/lib/pulp/imports` directory gets extracted to `/var/lib/pulp/`. You can empty the `/var/lib/pulp/imports` directory after the successful import.
```
Avoid future tense if possible. We should avoid using phrasal verbs per IBM SG. Active voice. "Empty" is more precise than "clear" I think. Split complex sentences.

Btw, can you try to rephrase the first sentence to use active voice as well?